source/extensions/common/dynamic_forward_proxy/dns_cache_impl.cc

```diff
@@ -422,6 +417,19 @@ void DnsCacheImpl::finishResolve(const std::string& host,
     return primary_host_it->second.get();
   }();
 
+  std::string details_with_maybe_trace = std::string(details);
+  if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.dfp_trace")) {
```

Comment:
oh I think I may not have been clear on what an option was in this case.  I meant to not do runtime guard, and instead have a non-runtime boolean option knob.  Basically I don't think it makes sense for non-mobile to have a trace in the response details header, so I was thinking we could have an undocumented runtime key that'd stick around permanently and be used just for mobile.

I'm fine with that and the config change, or only having the runtime guard - either wya
/waut

Reply:
Done. Updated the code to use a runtime option knob.